integrationtest/mapper/orm/src/test/java/org/hibernate/search/integrationtest/mapper/orm/bootstrap/BootstrapLogsIT.java

```diff
@@ -101,6 +103,27 @@ public void version() {
 		assertThat( Version.getVersionString() ).isEqualTo( expectedHibernateSearchVersion );
 	}
 
+	@Test
+	@TestForIssue(jiraKey = "HSEARCH-4503")
+	public void versionLoggingDisabled() {
+		String propertyKey = "org.hibernate.search.version";
+		String expectedHibernateSearchVersion = System.getProperty( propertyKey );
+		if ( expectedHibernateSearchVersion == null ) {
+			throw new IllegalStateException( "This test cannot be executed, because system property '"
+					+ propertyKey + "' was not defined." );
+		}
+
+		try ( SystemPropertyRestorer systemPropertyChange = SystemHelper.setSystemProperty( "jboss.log-versions", "false" ) ) {
```

Comment:
```suggestion
		try ( SystemPropertyRestorer systemPropertyChange = SystemHelper.setSystemProperty( HibernateOrmMapperSpiSettings.JBOSS_LOG_VERSIONS, "false" ) ) {
```

Reply:
Thanks, but actually that was on purpose.

We've been bitten in the past by tests using property key constants:

* sometimes we'd update a constant (either intentionally or by mistake) but forget to update documentation or to announce the backwards-incompatible change. The failing test would at least give us a heads-up in such case.
* or in other cases, the constant was not set to the value we intended, but the tests passed anyway, because, well, we'd use the same constant in the test and implementation. If we hard-code the property key in the test, this cannot happen.

So, yeah, I'd rather keep hardcoded strings in tests unless there's a specific reason to avoid that.